types/koa-generic-session/index.d.ts

```diff
@@ -37,6 +37,7 @@ declare namespace koaSession {
             maxAge?: number;
             secure?: boolean;
             httpOnly?: boolean;
+            sameSite?: string;
```

Comment:
Looks like it could be boolean or a string, maybe this is a better option?

```suggestion
            sameSite?: boolean | "lax" | "none" | "strict";
```

Reply:
TIL you can do that in TypeScript :) Thanks, I accepted your suggestion.

Reply:
Yeah! More on [that topic here](https://typescript-site-60.ortam.now.sh/play/index.html?#example/union-and-intersection-types)